pulp_container/app/viewsets.py

```diff
@@ -433,6 +455,30 @@ class ContainerDistributionViewSet(BaseDistributionViewSet):
     queryset = models.ContainerDistribution.objects.all()
     serializer_class = serializers.ContainerDistributionSerializer
 
+    @extend_schema(
+        description="Trigger an asynchronous delete task",
+        responses={202: AsyncOperationResponseSerializer},
+    )
+    def destroy(self, request, pk, **kwargs):
+        """
+        Delete a push repository with its distribution
```

Comment:
Delete a distribution. If a push repository is associated to it, delete it as well.